__main__.py

```diff
@@ -91,7 +92,7 @@ def main():
 
             for script in script_list:
                 if script.name:
-                    if args.l!=None:
+                    if args.l is not None:
```

Comment:
I am not sure of args.l
I have seen it coming in from acaa7076241a36cb754021f1d450522b32a66ae3
but the implementation may have something wrong.